profiling-ffi/src/exporter.rs

```diff
@@ -238,9 +218,8 @@ pub unsafe extern "C" fn ddog_prof_Exporter_Request_build(
 ///
 /// # Arguments
 /// * `exporter` - Borrows the exporter for sending the request.
-/// * `request` - Takes ownership of the request. Do not call `ddog_prof_Request_drop` nor
-///               `ddog_prof_Exporter_SendResult_drop` on `request` after calling
-///               `ddog_prof_Exporter_send` on it.
+/// * `request` - Takes ownership of the request. Do not call `ddog_prof_Request_drop` on
```

Comment:
Minor: Fixing namespace :)

```suggestion
/// * `request` - Takes ownership of the request. Do not call `ddog_prof_Exporter_Request_drop` on
```